pkgs/top-level/all-packages.nix

```diff
@@ -23650,6 +23650,10 @@ in
     };
   };
 
+  nml = callPackage ../games/openttd/nml.nix {};
```

Comment:
I am thinking whether the packages should not be named `openttd-nml`and `openttd-grfcodec` instead of `nml` and `grfcodec`, but no strong opinion here.